numba/tests/test_nrt.py

```diff
@@ -0,0 +1,174 @@
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+from numba import unittest_support as unittest
+from numba.runtime import rtsys
+from numba.config import PYVERSION
+
+
+class Dummy(object):
+    alive = 0
+
+    def __init__(self):
+        type(self).alive += 1
+
+    def __del__(self):
+        type(self).alive -= 1
+
+
+class TestNrtMemInfo(unittest.TestCase):
```

Comment:
If you don't want a test failure to ruin all subsequent tests, you could add `Dummy.alive = 0` in a setUp() method.